core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -1241,6 +1243,7 @@ private void processQuery(
         final Object statement = factoryCache.peek(queryText);
         if (statement == null) {
             final CompiledQuery cc = compiler.compile(queryText, sqlExecutionContext);
+            sqlExecutionContext.storeTelemetry(cc.getType());
```

Comment:
I hate to creep the scope here, but it would be super useful to capture execution site, e.g. PG wire, http etc